<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>正交汉字生成器 v0.1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="1,0">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>源文本</string>
      </property>
      <widget class="QPlainTextEdit" name="txtSrc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>366</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>目标文本</string>
      </property>
      <widget class="QPlainTextEdit" name="txtDest">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>366</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>95</height>
         </size>
        </property>
        <property name="title">
         <string>选择汉字库</string>
        </property>
        <widget class="QLineEdit" name="txtPath">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>130</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btnExplore">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>25</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>…</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnRead">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>115</x>
           <y>60</y>
           <width>60</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>读取</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtCount">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>60</y>
           <width>60</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>60</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>字数</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>操作</string>
        </property>
        <widget class="QPushButton" name="btnTransfer">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>转换</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>120</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>笔画：     -</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtMin">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>20</y>
           <width>30</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtMax">
         <property name="geometry">
          <rect>
           <x>115</x>
           <y>20</y>
           <width>30</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>22</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
